src/recharge.iced

```diff
@@ -101,8 +101,10 @@ exports.Runner = class Runner extends Base
 
     fee = @fee_estimator { tx: tx }
 
-    if (@debug)
-      console.log('Expected total fee:', num*@min_amount() + fee)
+    if @debug()
```

Comment:
why are we using the hardcoded transaction sizes for the inner transactions but the computed ones for the outer transaction?